Internal only patch to change the color of pie charts drawn by
Horizon to blue.  This patch is a recommendation from the UX design
team and will not be committed upsteam.

--- horizon-2013.2.3/horizon/static/horizon/js/horizon.d3piechart.js.orig  2014-06-11 13:19:35.874027396 -0600
+++ horizon-2013.2.3/horizon/static/horizon/js/horizon.d3piechart.js       2014-06-11 13:19:42.692287410 -0600
@@ -17,9 +17,13 @@
   r: 45,
   bkgrnd: "#F2F2F2",
   frgrnd: "#006CCF",
-  full: "#D0342B",
-  nearlyfull: "orange",
-
+  full: "#006CCF",
+  nearlyfull: "006CCF",
+  
+  // frgrnd: "#006CCF",
+  // full: "#D0342B",
+  // nearlyfull: "orange",
+  
   init: function() {
     var self = this;
 
